src/main/java/com/denizenscript/depenizen/bukkit/objects/mythicmobs/MythicSpawnerTag.java

```diff
@@ -0,0 +1,341 @@
+        return valueOf(name, null);
+    }
+
+    @Fetchable("mythicspawner")
+    public static MythicSpawnerTag valueOf(String string, TagContext context) {
+        if (string == null) {
+            return null;
+        }
+        try {
+            string = string.replace("mythicspawner@", "");
+            if (!MythicMobsBridge.isMythicSpawner(string)) {
+                return null;
+            }
+            return new MythicSpawnerTag(MythicMobsBridge.getMythicSpawner(string));
+        }
+        catch (Exception e) {
+            return null;
+        }
+    }
+
+    public static boolean matches(String string) {
+        return valueOf(string) != null;
+    }
+
+    public MythicSpawnerTag(MythicSpawner spawner) {
+        if (spawner != null) {
+            this.spawner = spawner;
+        }
+        else {
+            Debug.echoError("Mythic Spawner referenced is null!");
+        }
+    }
+
+    public MythicSpawner getSpawner() {
+        return spawner;
+    }
+
+    private String prefix;
+    MythicSpawner spawner;
+
+    @Override
+    public ObjectTag setPrefix(String prefix) {
+        this.prefix = prefix;
+        return this;
+    }
+
+    @Override
+    public ObjectTag getObjectAttribute(Attribute attribute) {
+        return tagProcessor.getObjectAttribute(this, attribute);
+    }
+
+    @Override
+    public String getPrefix() {
+        return prefix;
+    }
+
+    @Override
+    public String debug() {
+        return prefix + "='<A>" + identify() + "<G>' ";
+    }
+
+    @Override
+    public boolean isUnique() {
+        return true;
+    }
+
+    @Override
+    public String getObjectType() {
+        return "MythicSpawner";
+    }
+
+    @Override
+    public String identify() {
+        return "mythicspawner@" + spawner.getName();
+    }
+
+    @Override
+    public String identifySimple() {
+        return identify();
+    }
+
+    @Override
+    public String toString() {
+        return identify();
+    }
+
+    public static void registerTag(String name, TagRunnable.ObjectInterface<MythicSpawnerTag> runnable, String... variants) {
+        tagProcessor.registerTag(name, runnable, variants);
+    }
+    public static ObjectTagProcessor<MythicSpawnerTag> tagProcessor = new ObjectTagProcessor<>();
+
+    public static void registerTags() {
+
+        // <--[tag]
+        // @attribute <MythicSpawnerTag.name>
+        // @returns ElementTag
+        // @plugin Depenizen, MythicMobs
+        // @description
+        // Returns the name of the MythicSpawner.
+        // -->
+        registerTag("name", (attribute, object) -> {
+            return new ElementTag(object.getSpawner().getName());
+        });
+
+        // <--[tag]
+        // @attribute <MythicSpawnerTag.cooldown>
+        // @returns DurationTag
+        // @plugin Depenizen, MythicMobs
+        // @mechanism MythicSpawnerTag.cooldown
+        // @description
+        // Returns an ElementTag(Number) of the MythicSpawner's cooldown.
```

Comment:
That `an ElementTag(Number) of` is wrong and even if it wasn't still shouldn't be in the description part.